### PR TITLE
GH-1336 MARSHALLER_PERSIST_GRAPH_FIX

### PR DESCRIPTION
**Pull request by mlzummo**
Allows the graph to persist and recreate new vertexes/edges

Fixes GH-1336

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith @jbrundage Please Review (i can show you what this fixes ... gordon if you have a better solution let me know)

**Comment by GordonSmith** [comment]
@mlzummo - i am pretty sure this will work fine (as the Vertex + Edge are loaded if the Graph has been loaded) - but it might be simpler to just add the following to Graph.js:

```
Graph.prototype.Vertex = Vertex;
Graph.prototype.Edge = Edge;
```

And the we can simply do:
```
var v = new graph.Vertex();
```
or
```
var v = new Graph.prototype.Vertex();
```

**Comment by mlzummo** [comment]
@GordonSmith but how will we access this graph object from within GraphMapping.prototype.doMapAlll which is where we need this?

**Comment by mlzummo** [comment]
i think i see a way one sec

**Comment by mlzummo** [comment]
@GordonSmith made suggested changes

**Comment by mlzummo** [comment]
@GordonSmith can you pull this?

File: src/graph/Graph.js [hunk 1]
```diff
@@ -558,9 +561,6 @@
         edgeElements.enter().append("g")
             .attr("class", "graphEdge")
             .style("opacity", 1e-6)
-            .on("click.selectionBag", function (d) {
-                context._selection.click(d, d3.event);
-            })
```
Comment (GordonSmith) [inline src/graph/Graph.js]: Why is this removed?

**Comment by GordonSmith** [comment]
Can you clean up the other places where we were using the global Vertex and global Edge?

File: src/graph/Graph.js (hunk 1 above)
Comment (mlzummo) [inline src/graph/Graph.js]: idk i was getting something weird with that ... where it looked like i needed it to be removed .... but im going to put it back

**Comment by mlzummo** [comment]
@GordonSmith other places in the marshaller or in general throughout the lib?

**Comment by GordonSmith** [comment]
These items can be removed and then replaced with the new declarations:
https://github.com/mlzummo/Visualization/blob/MARSHALLER_PERSIST_GRAPH_FIX/src/marshaller/HipieDDL.js#L9-L10

File: src/graph/Graph.js [hunk 2]
```diff
@@ -558,8 +561,8 @@
         edgeElements.enter().append("g")
             .attr("class", "graphEdge")
             .style("opacity", 1e-6)
-            .on("click.selectionBag", function (d) {
```
Comment (mlzummo) [inline src/graph/Graph.js]: this is why i removed it in the first place idk why its red over here

**Comment by mlzummo** [comment]
@GordonSmith should be ready to pull now
